draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -55,10 +55,10 @@ secured against modification using JSON Web Signature (JWS) [@!RFC7515] digital
 signatures. A consumer of a signed JWT that has checked the
 signature can safely assume that the contents of the token have not been
 modified.  However, anyone receiving an unencrypted JWT can read all the
-claims and likewise, anyone with the decryption key receiving an encrypted JWT
+claims. Likewise, anyone with the decryption key receiving a signed and encrypted JWT
```

Comment:
The 'signed' part is besides the point (we did say only 'unencrypted' in the previous sentence) because that's what matters here.

Reply:
```suggestion
claims. Likewise, anyone with the decryption key receiving encrypted JWT
```